client/core/core.go

```diff
@@ -2445,7 +2443,7 @@ func (c *Core) prepareTrackedTrade(dc *dexConnection, form *TradeForm, crypter e
 	c.syncOrderPlaced(ord.ID())
 
 	// Send a low-priority notification.
-	corder, _ := tracker.coreOrder()
+	corder := tracker.coreOrder()
```

Comment:
Just noticing that since the tracker is being constructed in this method, you could use `coreOrderInternal` if you do this just prior to adding it to `dc.trades`.